tests/ceph-benchmarking/tests/bundles/bionic-nautilus.yaml

```diff
@@ -4,15 +4,11 @@ applications:
     num_units: 1
     series: bionic
     charm: vault
+    options:
+      totally-unsecure-auto-unlock: True
```

Comment:
Would it not be possible to remove keystone and use the `zaza.openstack.charm_tests.vault.setup.auto_initialize_no_validation` ([example](https://opendev.org/x/charm-ovn-central/src/branch/master/src/tests/tests.yaml#L21)) instead? I am a bit skeptic to us using the `totally-unsecure-auto-unlock` configuration option and would hope we could remove it one day.

Reply:
I'll test that.